classes/article/SubmissionFileDAO.inc.php

```diff
@@ -64,6 +64,8 @@ function getGenreCategoryMapping() {
 	function fromRow($row) {
 		if (isset($row['artwork_file_id']) && is_numeric($row['artwork_file_id'])) {
 			return parent::fromRow($row, 'SubmissionArtworkFile');
+		} elseif (isset($row['supplementary_file_id']) && is_numeric($row['supplementary_file_id'])) {
```

Comment:
I thought we got rid of this concept of supplementary files in OJS/OMP 3.0, in favor of a more flexible solution using genres... ArtworkFile makes sense to me, but supplementary file could be anything that's not the submission file itself... that's right?